src/app/modules/map/core/classes/regobs-geohazard-marker.ts

```diff
@@ -1,227 +1,42 @@
-import * as L from 'leaflet';
+import PictureMarkerSymbol from '@arcgis/core/symbols/PictureMarkerSymbol';
 import { GeoHazard } from '../../../../core/models/geo-hazard.enum';
 
 /**
  * TODO: Should we use a map service to show observations instead of using this?
  */
-export class RegobsGeoHazardMarker extends L.DivIcon {
+export class RegobsGeoHazardMarker {
```

Comment:
Her er det mye kode som ikke blir brukt. Trenger dette å være en egen klasse lengre eller kunne man like gjerne flytta de to funksjonene som brukes inn i MapItemMarker?